src/Symfony/Bundle/SecurityBundle/composer.json

```diff
@@ -17,6 +17,7 @@
     ],
     "require": {
         "php": ">=5.3.9",
+        "ext-xml": "*",
```

Comment:
I would keep it as using the bundle without DI is probably not a use case anyway.